apps/fluent-tester/src/E2E/common/BasePage.win.ts

```diff
@@ -2,7 +2,7 @@ const DUMMY_CHAR = '';
 export const COMPONENT_SCROLL_COORDINATES = { x: -0, y: -100 }; // These are the offsets. Y is negative because we want the touch to move up (and thus it scrolls down)
 
 /* Win32/UWP-Specific Selector. We use this to get elements on the test page */
-export function By(identifier: string): WebdriverIO.Element {
+export function By(identifier: string) {
```

Comment:
This works? I remember having trouble converting this one and ended up doing this:

```
export async function By(identifier: string): Promise<WebdriverIO.Element> {
  const element = await $('~' + identifier);
  return element;
}
```
but what you've done looks cleaner.

Reply:
Yes, this works :)

Do you remember the issue you had with it?